setup.py

```diff
@@ -11,15 +11,14 @@
     history = history_file.read()
 
 requirements = [
-    "gptcache<=0.1.35",
-    "langchain<=0.0.226",
+    "langchain>=0.2.6",
```

Comment:
should this be strictly greater than?

Reply:
or...do we even need to pin a version?

Reply:
I agree with you, no need to pin the version.